DRAFT.md

```diff
@@ -1051,28 +1072,19 @@ constexpr indirect& operator=(const indirect& other);
 
 1. _Mandates_: `T` is a complete type.
 
-2. _Effects_: If `addressof(other) == this` is `true`, there are no effects.
+2. _Effects_: If `addressof(other) == this` is `true`, there are no effects. Otherwise:
 
-  If `allocator_traits<allocator_type>::propagate_on_container_copy_assignment` is
-  `true` then the allocator needs updating.
+    2.1. The allocator needs updating if `allocator_traits<Allocator>::propagate_on_container_copy_assignment` is `true`.
 
-  If `other` is valueless, `*this` becomes valueless and the owned object in
-  `*this`, if any, is destroyed using `allocator_traits<allocator_type>::destroy`
-  and then the storage is deallocated.
+    2.2. If `other` is valueless, `*this` becomes valueless and the owned object in `*this`, if any, is destroyed using `allocator_traits<Allocator>::destroy` and then the storage is deallocated.
 
-  Otherwise, if `alloc == other.alloc` is `true` and `*this` is not valueless,
-  `*other` is assigned to the owned object.
+    2.3. Otherwise, if `alloc == other.alloc` is `true` and `*this` is not valueless, `*other` is assigned to the owned object.
 
-  Otherwise a new owned object is constructed in `*this` using
-  `allocator_traits<allocator_type>::construct` with the owned object from
-  `other` as the argument, using either the allocator in `*this` or the
-  allocator in `other` if the allocator needs updating. The previously owned
-  object in `*this`, if any, is destroyed using
-  `allocator_traits<allocator_type>::destroy` and then the storage is
-  deallocated.
+    2.4. Otherwise a new owned object is constructed in `*this` using `allocator_traits<Allocator>::construct` with the owned object from `other` as the argument, using either the allocator in `*this` or the allocator in `other` if the allocator needs updating.
+
+    2.5. The previously owned object in `*this`, if any, is destroyed using `allocator_traits<Allocator>::destroy` and then the storage is deallocated.
 
-  If the allocator needs updating, the allocator in `*this` is replaced with a
-  copy of the allocator in `other`.
+    2.6. If the allocator needs updating, the allocator in `*this` is replaced with a copy of the allocator in `other`.
```

Comment:
This sentence is unclear what "replaced with a copy of the allocator in `other`" means.
It could be copy-assignment or destruct+copy-construct in-place.

If this means that `alloc = other.alloc` then we should say "the allocator in `other` is copy-assigned to `alloc`."